src/ARAnchorManager.js

```diff
@@ -0,0 +1,93 @@
+// import { EventEmitter } from 'events';
```

Comment:
What's the issue with events? We may need to do an extra line due to how destructuring works during import (named exports) vs on an object with properties (traditional destructuring)

```js
import events from 'events';
const { EventEmitter } = events;
```

Although would it make more sense to use THREE's [EventDispatcher](https://threejs.org/docs/#api/core/EventDispatcher)? Maybe more consistent with other THREE works, since users are already going to have that dependency anyway

Reply:
Ok.